### PR TITLE
feature:modify client api

### PR DESCRIPTION
**Pull request by yunfeiyanggzq**
Signed-off-by: yunfeiyangbuaa <yunfeiyang@buaa.edu.cn>

### Ⅰ. Describe what this PR did
in this pr,
I do these things:

1. make dfget request url int the format `api/v1/*`
2. modify `config` to `request`
3. and make client interface be the sub-interface of `CommonAPIClient`  

### Ⅱ. Does this pull request fix one issue?

refrence #1082 #1065

### CODE REVIEW

**Comment by Starnop** [comment]
LGTM.

File: client/interface.go [hunk 1]
```diff
@@ -25,9 +25,24 @@ import (
...
 
 // PreheatAPIClient defines methods of Container client.
 type PreheatAPIClient interface {
-	PreheatCreate(ctx context.Context, config *types.PreheatCreateRequest) (*types.PreheatCreateResponse, error)
+	PreheatCreate(ctx context.Context, request *types.PreheatCreateRequest) (*types.PreheatCreateResponse, error)
+}
+
+// PeerAPIClient defines methods of peer related client.
+type PeerAPIClient interface {
+	PeerCreate(ctx context.Context, request *types.PeerCreateRequest) (*types.PeerCreateResponse, error)
```
Comment (Starnop) [inline client/interface.go]: I think we should use the named variables for interface methods.

Comment (yunfeiyanggzq) [reply client/interface.go]: Done

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**